### PR TITLE
Web interface 2

### PR DESCRIPTION
**Pull request by feliciaan**
A couple of things changed
 - [x] Added GitHub OAuth2 as authentication
 - [x] Updated fcgi script
 - [x] now using SQLAlchemy as ORM
 - [x] moved settings to the config
 - [x] added logger

TODOs:
 - [x] update documentation
 - [x] starting the server (init.d or supervisor.d)
 - [x] run PEP-8
 - [x] add other people to the seed file and slack token through the app.config or so
 - [ ] unit tests
 - [x] better response in OAuth when the user is not in the list

### CODE REVIEW

**Comment by Silox** [comment]
And why exactly do we authenticate to Github and not to our own memberlist?

File: Web/slotmachien.fcgi [hunk 1]
```diff
@@ -1,14 +1,23 @@
+#!/Users/feliciaan/Documents/ZeusWPI/slotmachien/Web/env/bin/python
+import sys # insert path
+sys.path.insert(0, '/Users/feliciaan/Documents/ZeusWPI/slotmachien/Web/env/lib/python2.7/site-packages')
```
Comment (Silox) [inline Web/slotmachien.fcgi]: What is this?

File: Web/utils.py [hunk 2]
```diff
@@ -69,23 +66,29 @@ def clean_process(self):
...
         if not self.process or self.process.poll() or self.stopped:
-            print("DEAD")
-            self.create()
+            logger.info("Java process went down")
+            return False
+        return True
 
     def stdin(self):
-        self.check_alive()
+        if not self.check_alive():
```
Comment (Silox) [inline Web/utils.py]: Move all of this into a decorator and annotate every method using this with that decorator

**Comment by feliciaan** [comment]
@Silox it's better to use GitHub now, because it's unclear how to use kerberos outside of Zeus.

**Pull request merged**